freqtrade/optimize/hyperopt_loss_calmar.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+from pandas import DataFrame, Series
+from freqtrade.optimize.hyperopt import IHyperOptLoss
+import numpy as np
+from scipy.stats import norm
+
+NB_SIMULATIONS = 1000
+SIMULATION_YEAR_DURATION = 3
+HIGH_NUMBER = 100000
```

Comment:
`MAX_LOSS = 10000` is used in `hyperopt.py` for this purpose, this should probably be placed in the constants or imported from `freqtrade.optimize.hyperopt` directly.